internal-webapps/org.wso2.bfsi.authentication.endpoint/src/main/webapp/cookie_policy.jsp

```diff
@@ -0,0 +1,240 @@
+                              </td>
+                              <td>
+                                  <p>Session</p>
+                              </td>
+                          </tr>
+                          <tr>
+                              <td>
+                                  <p>requestedURI</p>
+                              </td>
+                              <td>
+                                  <p>The URI you are accessing.</p>
+                              </td>
+                              <td>
+                                  <p>Session</p>
+                              </td>
+                          </tr>
+                          <tr>
+                              <td>
+                                  <p>current-breadcrumb</p>
+                              </td>
+                              <td>
+                                  <p>To keep your active page in session in order to give you a good user experience.</p>
+                              </td>
+                              <td>
+                                  <p>Session</p>
+                              </td>
+                          </tr>
+                          </tbody>
+                      </table>
+                  </section>
+
+                  <section>
+                      <h2 id="disclaimer">Disclaimer</h2>
+                      <ol>
+                          <li>This cookie policy is only for the illustrative purposes of the product WSO2 Open Banking. The content in the policy is technically correct at the time of the product shipment. The organization which runs this WSO2 Open Banking instance has full authority and responsibility with regard to the effective Cookie Policy.</br><br></li>
+                          <li>WSO2, its employees, partners, and affiliates do not have access to and do not require, store, process or control any of the data, including personal data contained in WSO2 Open Banking. All data, including personal data is controlled and processed by the entity or individual running WSO2 Open Banking.  WSO2, its employees partners and affiliates are not a data processor or a data controller within the meaning of any data privacy regulations.  WSO2 does not provide any warranties or undertake any responsibility or liability in connection with the lawfulness or the manner and purposes for which WSO2 Open Banking is used by such entities or persons. </li>
+                      </ol>
+                  </section>
+              </div>
+              <!-- /Costomizable content -->
+          </div>
+      </div>
+      <!-- /content -->
+
+  </div>
+        
+  <!-- footer -->
+  <jsp:include page="includes/footer.jsp"/>
+        
+  <script src="libs/jquery_3.5.0/jquery-3.5.0.js"></script>
+  <script src="libs/bootstrap_3.4.1/js/bootstrap.min.js"></script>
+
+  <script type="text/javascript" src="js/u2f-api.js"></script>
```

Comment:
What is the usage of this module?

Reply:
FIxed in https://github.com/wso2-extensions/bfsi-consent-management/pull/8/commits/95075fd154dc083e2d513679c151515167cec132